static/docs/command-reference/update.md

```diff
@@ -39,6 +39,12 @@ to learn how to "update" fixed-revision imports.
 
 ## Options
 
+- `--rev` - specific
```

Comment:
see the note in the description above ... we should probably update it as well? does `--rev` override `dvc import --rev`, for example?

Reply:
Agree. I think we should revise the last paragraph in the description "... when the --rev (revision) option of dvc import has been used to create an import stage..." and/or add a new one explaining the relationship/interactions/cases when using --rev options in dvc import and then in dvc update again.
> As commented on https://github.com/iterative/dvc.org/pull/890#pullrequestreview-337783511

Reply:
https://github.com/iterative/dvc.org/pull/890#discussion_r363048562

Reply:
done by 82c8b5c, can you please re-check?